src/client/config/menuItems.ts

```diff
@@ -137,6 +137,7 @@ const myChaptersMenuItems: ChapterMenuItem[] = [
     id: Chapters.PARKEREN,
     to: AppRoutes.PARKEREN,
     profileTypes: ['private', 'private-commercial', 'commercial'],
+    isAlwaysVisible: true,
```

Comment:
Parkeren heeft ook nog checks op of iemand in Amsterdam woont, hiermee wordt de pagina helemaal open gezet. Is dat wel wenselijk?

Reply:
Klopt idd. Ik fix 'm